src/main/resources/application.conf

```diff
@@ -185,8 +185,8 @@ mantis {
       }
 
       # Enabled JSON-RPC APIs over the JSON-RPC endpoint
-      # Available choices are: eth, web3, net, personal, test, daedalus, iele
-      apis = "eth,web3,net,personal,daedalus,debug"
+      # Available choices are: eth, web3, net, personal, test, daedalus, iele, qa
+      apis = "eth,web3,net,personal,daedalus,debug,qa"
```

Comment:
shouldn't section about available consensus/pow algorithms be updated as well?
shouldn't config in universal be updated for sake of documentation?